test/integration/whats_new_test.rb

```diff
@@ -24,8 +24,8 @@ class WhatsNewTest < ActionDispatch::IntegrationTest
     login_as create(:gds_editor)
     get admin_whats_new_path
 
-    if I18n.t("admin.whats_new.show_banner")
-      assert_select ".gem-c-phase-banner"
+    if I18n.t("admin.whats_new.show_banner") || I18n.t("admin.feedback.show_banner")
+      assert_select ".gem-c-phase-banner", count: 1
     else
       assert_select ".gem-c-phase-banner", count: 0
     end
```

Comment:
This test isn't all that great, because it doesn't stub/override the values of `I18n.t("admin.whats_new.show_banner")` or `I18n.t("admin.feedback.show_banner")`.

But again, this is a pre-existing smell – not something you've introduced. It's also not exactly mission critical functionality. So (wearing my pragmatic hat) I think this is probably okay as is.